packages/mantine/src/components/themedLayoutV2/title/index.tsx

```diff
@@ -52,21 +51,10 @@ export const ThemedTitleV2: React.FC<RefineLayoutThemedTitleProps> = ({
           ...wrapperStyles,
         }}
       >
-        <Text
-          lh={0}
-          fz="inherit"
-          color={theme.colorScheme === "dark" ? "brand.5" : "brand.6"}
-        >
+        <Text lh={0} fz="inherit">
```

Comment:
```suggestion
        <Text lh={0} fz="inherit" c="primary">
```

icon color should be primary

https://github.com/refinedev/refine/assets/23058882/4eb9e4ef-5b4f-4fcf-9a93-039420c9e11f